x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/suggestion_panel.test.tsx

```diff
@@ -232,6 +251,8 @@ describe('suggestion_panel', () => {
       },
     ] as Suggestion[]);
 
+    (mockVisualization.toPreviewExpression as jest.Mock).mockReturnValueOnce(undefined);
+    (mockVisualization.toPreviewExpression as jest.Mock).mockReturnValueOnce('test | expression');
```

Comment:
I don't see any tests in the suggestion panel that are directly related to the temporary suggestions, is that something you can do?

Reply:
Good point, I added some tests. I also noticed a bug which I fixed (no "Current" panel if the current visualization can't render a preview, e.g. tables)